<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>791</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../4.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="addMovie">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ADD MOVIE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeMovie">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>REMOVE MOVIE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="add">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-5</y>
     <width>791</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../5.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>185</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="type">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="category">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newMovie">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>320</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="removeLabel">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-5</y>
     <width>791</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../6.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="moviesForRemove">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <zorder>removeLabel</zorder>
  <zorder>add</zorder>
  <zorder>label</zorder>
  <zorder>name</zorder>
  <zorder>path</zorder>
  <zorder>type</zorder>
  <zorder>category</zorder>
  <zorder>newMovie</zorder>
  <zorder>back</zorder>
  <zorder>removeMovie</zorder>
  <zorder>addMovie</zorder>
  <zorder>moviesForRemove</zorder>
  <zorder>removeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
